src/Agent/NewRelic/Agent/Core/Aggregators/LogEventAggregator.cs

```diff
@@ -11,6 +11,7 @@
 using NewRelic.Agent.Core.Time;
 using NewRelic.Agent.Core.WireModels;
 using NewRelic.Collections;
+using NewRelic.Core.Logging;
```

Comment:
nitpicky: This using is not needed.

Reply:
Aww dang! It's leftover from the debugging log message I added then removed. Good catch